eos/statistics/log-prior_TEST.cc

```diff
@@ -284,6 +284,24 @@ class LogPriorTest :
                 TEST_CHECK_NEARLY_EQUAL(inverse_cdf(poisson_prior, param, 0.95) * k, 16.9622192357219,   eps);
             }
 
+            // Transform prior (w/ infinite support)
```

Comment:
Hm, this is not true, right? This prior has finite support. I would remove the ``(w/ infinite support)``.